WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -293,6 +298,36 @@ public void run() {
         }
     }
 
+    private void handleAppBannerDeepLink() {
```

Comment:
Currently this simply handles app banner deep links.

App Banner deep links are sent as `
intent://post/#Intent;scheme=wordpress;package=org.wordpress.android;end` and have the format:
```
<data
                    android:host="stats"
                    android:scheme="wordpress" />
```

Whereas url-intercepting-deep-links are simple urls and our intent filters are in this format:
```
<data
                    android:host="wordpress.com"
                    android:pathPattern="/read/feeds/.*/posts/.*"
                    android:scheme="https" >
                </data>
```

(The reason we set the URL as `intent://...` is because that opens the Play Store if the app isn't installed. If it were the normal `https://wordpress.com/...` link it would simply continue on in the browser.

Reply:
But I do agree with you. Any changes that allow for generalized deep links handling we should reflect it properly in the name when it happens.

Reply:
Ohh, thanks for clarification! 👍